dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -575,6 +583,8 @@ def __init__(
         self,
         fargate_scheduler=False,
         fargate_workers=False,
+        fargate_spot=False,
```

Comment:
Methinks it's a bug. In turn, the param value from the config file won't be loaded.

Either change `fargate_spot=False` to `fargate_spot=None` or change the logic altogether. Later in the code you do the following check:

```
if self._fargate_spot is None:
    self._fargate_spot = self.config.get("fargate_spot")
``` 

Since `fargate_spot` is defined as `False`, the param value from the config file won't be loaded. To reproduce:

```
# define FARGATE_SPOT in cloudprovider.yaml

cluster = FargateCluster()
print(cluster._fargate_spot)
```